### ISSUE TITLE
Doesn't work for Ducky 1...

### ISSUE THREAD

**Issue by BopItFreak** [Vidalee/DuckyVisual#4]
Downloaded the program, got this: 
![image](https://user-images.githubusercontent.com/41522720/93399914-8b825980-f84c-11ea-8c9a-798a012fe88a.png)

**Comment by BopItFreak** [comment]
Update: I investigated this issue a bit more, and I found that another device was causing that exception. With that out of the way however, I got "The binary key cannot have an odd number of digits" error. No idea what's going on... Commenting out that error gives more errors.

**Comment by Vidalee** [comment]
Yeah it only works for the Shine 6 since I only have this one, thus I can only reverse this one. The product ID is hardcoded here, you can to change it for you own keyboard and see if it works, but probably not. https://github.com/Vidalee/DuckyVisual/blob/master/src/Program.cs#L33

**Comment by BopItFreak** [comment]
Yeah I did that, no luck.
